ts/features/it-wallet/screens/credential/ItwCredentialDetailsScreen.tsx

```diff
@@ -0,0 +1,101 @@
+import React from "react";
+import { RouteProp, useRoute } from "@react-navigation/native";
+import {
+  Banner,
+  BlockButtonProps,
+  FooterWithButtons,
+  VSpacer
+} from "@pagopa/io-app-design-system";
+import { ScrollView } from "react-native-gesture-handler";
+import { SafeAreaView, View } from "react-native";
+import { constNull } from "fp-ts/lib/function";
+import ItwCredentialCard from "../../components/ItwCredentialCard";
+import { IOStyles } from "../../../../components/core/variables/IOStyles";
+import BaseScreenComponent from "../../../../components/screens/BaseScreenComponent";
+import I18n from "../../../../i18n";
+import { emptyContextualHelp } from "../../../../utils/emptyContextualHelp";
+import { ItwParamsList } from "../../navigation/ItwParamsList";
+import ItwCredentialClaimsList from "../../components/ItwCredentialClaimsList";
+import { StoredCredential } from "../../store/reducers/itwCredentialsReducer";
+
+export type ItwCredentialDetailsScreenNavigationParams = {
+  credential: StoredCredential;
+};
+
+type ItwCredentialDetailscreenRouteProps = RouteProp<
+  ItwParamsList,
+  "ITW_CREDENTIAL_DETAILS"
+>;
+
+/**
+ * Renders a preview screen which displays a visual representation and the claims contained in the credential.
+ */
+const ItwCredentialDetailsScreen = () => {
+  const route = useRoute<ItwCredentialDetailscreenRouteProps>();
+  const { credential } = route.params;
+  const bannerViewRef = React.createRef<View>();
+
+  /**
+   * Content view which asks the user to confirm the issuance of the credential.
+   * @param data - the issuance result data of the credential used to display the credential.
+   */
+  const ContentView = ({ data }: { data: StoredCredential }) => {
+    const presentationButton: BlockButtonProps = {
+      type: "Solid",
+      buttonProps: {
+        label: I18n.t(
+          "features.itWallet.presentation.credentialDetails.buttons.qrCode"
+        ),
+        accessibilityLabel: I18n.t(
+          "features.itWallet.presentation.credentialDetails.buttons.qrCode"
+        ),
+        onPress: () => null
```

Comment:
Should we add a navigation to the missing feature screen?

Reply:
There is already a task on jira related to the missing screen. I would then like to manage everything in a single PR / activity.

Reply:
All right, the activity is to add it where it's missing but I thought we could start using it already on new screens since the missing screen is already ready and added to the navigator. Up to you anyway, no big deal.